.github/workflows/product-tests-specific-environment.yml

```diff
@@ -14,8 +14,7 @@ env:
 jobs:
   product-tests-specific-environment1:
     runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
+    name: environment 1
```

Comment:
May be remove this change, on the tests on this PR there are two hung tests.

Reply:
It's the same thing as this:
https://github.com/prestodb/presto/pull/17384/files#r817057495

If something is hanging, then setting a shorter timeout (keeping in mind how long the task should run) would be a better approach.